ex13.py

```diff
@@ -0,0 +1,13 @@
+from sys import argv
```

Comment:
request - http library
wxPython - gui toolkit for python
SQLAlchemy - database library
BeautifulSoup - xml and html parsing library
 NumPy - math functionalities to python
sys - provides access to some variables used or maintained by the interpreter and to functions